### PR TITLE
Magic pod 0.54.1

### PR DESCRIPTION
**Pull request by nkns165**
TagCheck

There is already another step named "Magic Pod UI test", but we'd like to add drastic change to input parameters and cannot keep backward compatibility. So we will announce the existing step as deprecated (I will submit another PR).

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by nkns165** [comment]
CI check looks failed.  Can we have any information about the failure?

**Comment by non-binary** [comment]
The issue is that you need to create a step-info.yml file with `maintainer: community`. Here's an example from when I did this in my step: https://github.com/non-binary/bitrise-steplib/commit/cc28eab3890019f55517006d36c232fac2c391b4.

**Comment by nkns165** [comment]
Thank you, updated.  But there is still a problem?

**Comment by non-binary** [comment]
I had to trigger the build from the API (I am a bitrise employee). It now returned green. :)

File: steps/magic-pod/0.54.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+website: https://github.com/magic-Pod/bitrise-step-magicpod-uitest
+source_code_url: https://github.com/magic-Pod/bitrise-step-magicpod-uitest
+support_url: https://github.com/magic-Pod/bitrise-step-magicpod-uitest/issues
+published_at: 2020-06-02T22:24:46.286263+09:00
+source:
+  git: https://github.com/Magic-Pod/bitrise-step-magic-pod.git
+  commit: 3503680420460bbd2ab75446c2f6da2767e19df3
+project_type_tags:
+- ios
+- android
```
Comment (lpusok) [inline steps/magic-pod/0.54.1/step.yml:15]: Do you support multi platform apps like flutter and react-native? Not mandatory but could recommend to more users if such.

Comment (nkns165) [reply steps/magic-pod/0.54.1/step.yml:15]: Thank you for recommendation.  I think our service can handle such multi platform apps, but not especially customized for them.  We will think of such tags in the next version.

**Pull request merged**